### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by newellp88** [ejbrow/date-a-scientist#1]
## Rubric Score
 
### Criteria 1: Valid Python Code
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ Code runs without errors.
 
### Criteria 2: Exploration of Data
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ Data is thoroughly explored to rationalize the selection of features to answer the questions presented. It’s generally a good idea to check to see if features are correlated to the target value before proceeding, good work. Also, strong use of preprocessing, scaling, and mapping.
 
### Criteria 3: Machine Learning Techniques Used Correctly
* _Score Level:_ 3 (Meets Expectations)
* _Comment(s):_ Algorithms deployed are used correctly, although `sklearn.metrics.classification_report()` would have been a cleaner way to get all the accuracy scores you were looking for from your classification models.

### Criteria 4: Report: Are conclusions clear and supported by data?
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ Questions are clearly stated and the results of all 4 algorithms are shown and discussed. Conclusions are clear and based on the evidence provided. Despite not being overly conclusive, sometimes we just don’t have the all the data available that we want. Still, interesting questions and good setup.

### Criteria 5: Code Formatting
* _Score Level:_3 (Meets Expectations)
* _Comment(s):_ Code is relatively. Although, style guides usually suggest that module imports are at the top of the file not embedded within.
 
### Overall Score: 18/20
I enjoyed the “cat people” theme to your research, it’s one of the more interesting approaches to this project that I’ve seen. Although the data seemed rather stacked against you since most of the “has” columns were negative. Overall, it was an interesting take on the research and a good use of the skills covered in the course.
